### ISSUE TITLE
Inspect side-effects of disabling stack checking

### ISSUE THREAD

**Issue by LorenzMeier** [PX4/Firmware#1700]
This includes #1691. @davids5 @sjwilks These are the combined symptoms of disabling stack checking:

  * V1: microSD mount fails
  * V3: USB serial link (NuttX) is started, but never registers with the host (device doesn't list under /dev/tty.usb*)

I'm not 100% sure yet if this is timing or if we're missing something, but we clearly don't have everything under control yet.

**Comment by LorenzMeier** [comment]
@davids5 Ok, I've put the USB one to rest. Fixed via 18d756d. So only the microSD mount fail remains, and I wouldn't be terribly surprised if its timing again there.

**Comment by davids5** [comment]
@LorenzMeier Craig has confirmed that there will be HW heading my way at some point. I will dig into this bug then,

**Comment by LorenzMeier** [comment]
@sjwilks Could you try disabling the ms5611 driver startup? I'm wondering if the new SPI variant messes with the SD bus and the timing change made it visible.

**Comment by sjwilks** [comment]
While trying to workout why my F330 (PX4 V1) was going nuts since yesterday
I noticed that the SD card seems to have died completely. This probably
means my tests in the last two days weren't reliable. I have just replaced
the card, re-flashed master (via QGC) and I can read from the SD card. So
it looks like #1691 did indeed help. Prior to that I could constantly
read/not read based on the stack check setting so I think the SD failure
only just began.
